.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,6 +1,7 @@
 ---
 name: Feature request 
 about: Suggest adding a feature that is not yet in Spack  
+labels: feature, proposal
```

Comment:
I would only use feature here. If these two labels are too overlapping we could merge them.